components/PageControls.js

```diff
@@ -3,9 +3,9 @@ import { Button, Pagination } from 'react-bootstrap'
 
 function PageControls ({ limit, offset, result, setOffset, showSkipButtons = false, url }) {
   const { data: swrData = {}, error, isValidating, mutate } = result
-  const { records } = swrData
-  const hasRecords = records && records.data && records.data.length > 0
-  const lastOffset = hasRecords ? records.total - (records.total % limit) : 0
+  const { data: responseList } = swrData
+  const hasRecords = responseList && responseList.data && responseList.data.length > 0
```

Comment:
I think we started using this syntax haven't we?
```suggestion
  const hasRecords = responseList?.data?.length > 0
```